nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -270,6 +279,9 @@ private synchronized void initializeServer(final ProcessContext context) throws
         final int port = context.getProperty(PORT).asInteger();
         final SSLContextService sslService = context.getProperty(SSL_CONTEXT).asControllerService(SSLContextService.class);
 
+        final String rawTimeout = context.getProperty(TIMEOUT).getValue();
```

Comment:
These two lines can be simplified to:
{code}
final long timeout = context.getProperty(TIMEOUT).asTimePeriod(TimeUnit.MILLISECONDS);
{code}